src/picture_widget.rs

```diff
@@ -197,6 +200,21 @@ impl PictureWidgetData {
 		self.render_validity.invalidate();
 	}
 
+	pub fn toggle_antialias(&mut self) {
+		let aa = match self.antialiasing {
+			Antialias::Auto if self.img_texel_size < 4f32 => Antialias::Never,
```

Comment:
I just realized that the constant `4f32` appears twice in the code. We should introduce a const for it, like

```rust
const ANTIALIAS_TEXEL_THRESHOLD: f32 = 4f32;
```